sass/elements/_latest-posts.scss

```diff
@@ -0,0 +1,53 @@
+.wp-block-latest-posts {
+	@include light-on-dark;
+	font-family: $font-button;
+	font-weight: 700;
+	margin: spacing(2) 0;
+
+	li {
+		background: $grey--alt;
+		color: $white;
+		padding: spacing(2);
+		border-radius: $size__border-radius;
+	}
+
+	a {
+		@include font-size( 2.4 );
+		line-height: $font__line-height--small;
+		color: $white;
+		text-decoration: none;
+
+		&:hover,
+		&:focus,
+		&:visited {
```

Comment:
We have the `hover-state` mixin that handles this, so you can do 

```scss
@include hover-state {
    // your styles
}
```

Reply:
Updated to use the mixin.